sdk/core/azure-core/inc/azure/core/http/curl/curl connection_pool.hpp

```diff
@@ -27,6 +27,77 @@ namespace Azure { namespace Core { namespace Test {
 
 namespace Azure { namespace Core { namespace Http {
 
+  /**
+   * @brief The available options to set libcurl ssl options.
+   *
+   * @remark The SDK will map the enum option to libcurl's specific option. See more info here:
+   * https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
+   *
+   */
+  enum class CurlTransportSSLOptions
+  {
+    Default,
+    AllowBeast,
+    NoRevoke,
```

Comment:
Reading the docs, it looks like this option is only supported on Windows. What's the objective and scenario here?

Have we tested these on non-Windows platforms to see what happens?